src/sentry/api/issue_search.py

```diff
@@ -325,9 +363,9 @@ def expand_substatus_query_values(
             if search_filter.key.name == "substatus":
                 converted = convert_search_filter(search_filter, org)
                 new_value = converted.value.raw_value
-                status = GROUP_SUBSTATUS_TO_STATUS_MAP.get(
-                    new_value[0] if isinstance(new_value, list) else new_value
-                )
+                new_value_int = new_value[0] if isinstance(new_value, list) else new_value
+                assert isinstance(new_value_int, int), new_value_int
+                status = GROUP_SUBSTATUS_TO_STATUS_MAP[new_value_int]
```

Comment:
there's technically two new ways this could crash -- we could get a non-int or we could get a not valid value -- but I *think* either of those would have already crashed since it would have generated a `None` SearchValue?